hikari/impl/guild_chunker.py

```diff
@@ -200,17 +200,8 @@ def update(self, event: shard_events.MemberChunkEvent) -> None:
         self.last_received = time.utc_datetime()
 
 
-class StatefulGuildChunkerImpl(chunker.GuildChunker):
-    """Guild chunker implementation.
-
-    Parameters
-    ----------
-    app : hikari.traits.BotAware
```

Comment:
not gonna document the parameters anymore?

Reply:
This was too keep consistency with other components. Thinking about it again, I'm going to actually document it on the ones that are missing